test/fuzzing/find_fuzz.cpp

```diff
@@ -98,7 +98,8 @@ struct QueryFixture {
         qry_processor_,
         query_factory,
         blocks_query_factory,
-        logger::getDummyLoggerPtr());
+        logger::getDummyLoggerPtr(),
+        nullptr);
```

Comment:
I see that the new param is nowhere tested, isn't it?

Reply:
I'm not sure it can be and need to be tested. New param is for subscription to async events engine. This engine has it's own tests.